lib/ui/widgets/patchesSelectorView/patch_options_fields.dart

```diff
@@ -503,25 +505,59 @@ class _TextFieldForPatchOptionState extends State<TextFieldForPatchOption> {
                   ];
                 },
                 onSelected: (String selection) async {
-                  switch (selection) {
-                    case 'patchOptionsView.selectFilePath':
-                      final String? result = await FlutterFileDialog.pickFile();
-                      if (result != null) {
-                        controller.text = result;
-                        widget.onChanged(controller.text);
+                  // manageExternalStorage permission is required for file/folder selection
+                  // otherwise, the app will not complain, but the patches will error out
+                  // the same way as if the user selected an empty file/folder.
+                  // Android 10 and above requires the manageExternalStorage permission
```

Comment:
Since the `android:requestLegacyExternalStorage` value is [set to true](https://github.com/inotia00/revanced-manager/blob/4f8c559caf71a89ab6b596a8791941d85f52751d/android/app/src/main/AndroidManifest.xml#L29), there is no need to check this in Android 10.
(If you include Android 10 in the permission check, it will not work on Android 10)